rebasehelper/checker.py

```diff
@@ -52,29 +50,40 @@ def run_check(cls, results_dir):
         raise NotImplementedError()
 
 
-class Checker(object):
+class CheckersRunner(object):
     """
-    Class representing a process of checking final packages.
+    Class representing the process of running various checkers on final packages.
     """
 
-    def __init__(self, dir_name):
+    def __init__(self, dir_name=os.path.join(os.path.dirname(__file__), 'checkers')):
+        """
+        Constructor of a Checker class.
+
+        :param dir_name: Path to directory witch contains various checkers. By default the it looks for checkers inside
+        'checkers' subdirectory in the project's installation location.
+        :type dir_name: str
+        """
         self._injector_type = 'BaseChecker'
         self.plugin_classes = self.load_checkers(dir_name)
 
-    def __str__(self):
-        return "<Checker tool_name='{_tool_name}' tool={_tool}>".format(**vars(self))
-
-    def checker_find_injector(self, module):
+    def _checker_find_injector(self, module):
         injectors = []
         for n in dir(module):
             attr = getattr(module, n)
             if hasattr(attr, '__base__') and attr.__base__.__name__ == self._injector_type:
                 injectors.append(attr)
         return injectors
 
-    def load_checkers(self, dir_name):
+    def load_checkers(self, plugins_dir):
+        """
+        Load checker implementations from the given location.
+
+        :param plugins_dir: Path to directory from which to load the checkers.
+        :type plugins_dir: str
+        :return: Dictionary with names of the checkers and the actual checker objects.
+        :rtype: dict
+        """
         plugin_checkers = {}
-        plugins_dir = os.path.join(dir_name, 'checkers')
         for plugin in os.listdir(plugins_dir):
```

Comment:
plugins_dir is not defined. I think it should be mentioned in __init__ function.